devops/cts_exclude_filter/L0_GPU

```diff
@@ -1,4 +0,0 @@
-# CMPLRLLVM-61839
-multi_ptr
```

Comment:
This removal isn't a move...

Reply:
I don't know why GH interface doesn't show it, but I moved this test name to compfails as it fails on both L0 and OCL - https://github.com/intel/llvm/blob/d5da305317e2f094c7bd8fdac245fe36d2654611/sycl/cts_exclude_filter/compfails
I'll update the commit message

Reply:
My question was about actual content and not about GH interface displaying it. I've updated the PR's title, please check if it's ok with you.

Reply:
I'm okay with the change, thanks